mdflynn/rancid-tomatillos
# Testing/additional tests

## Is this a fix or a feature?
- Feature

## What is the change?
- Provides context to the rating numbers on the homepage
- Adds additional test scripts
- Formats all of the files 
- Adds a README

## Where should the reviewer start?
- App.js: Lines 100 and 103
- App.test.js: Lines 63-72 and 152-171 
- Movie.js: Lines 15 and 35
- README.md: Entire File

## Issues resolved
- #27
- #28

## How should this be tested?
- Clone down this repo
- `cd` into the cloned down directory
- Run `npm test` in the terminal
- Observe the tests pass
